### ISSUE TITLE
DHG dataset

### ISSUE THREAD

**Issue by weiyutao886** [multimodallearning/hand-gesture-posture-position#3]
Hello, I'm very interested in your experimental results on DHG data set. How do you divide the training set and test set? Can you share it?

**Comment by alexbgl** [comment]
I just uploaded scripts for training and testing on DHG. Please also see my answer to #2.
